src/client/game.cpp

```diff
@@ -3158,6 +3158,9 @@ PointedThing Game::updatePointedThing(
 	PointedThing result;
 	env.continueRaycast(&s, &result);
 	if (result.type == POINTEDTHING_OBJECT) {
+		if (!hud->can_draw_selectionindicator)
```

Comment:
for me the condition are useless, just do it unconditionnaly

Reply:
Done